src/iuse_actor.cpp

```diff
@@ -324,6 +325,9 @@ int iuse_transform::use( player &p, item &it, bool t, const tripoint &pos ) cons
 ret_val<bool> iuse_transform::can_use( const Character &p, const item &, bool,
                                        const tripoint & ) const
 {
+    if( need_dry && p.is_underwater() ) {
+        return ret_val<bool>::make_failure( _( "This item cannot be used while wet." ) );
```

Comment:
Minor thought, current phrasing may be confused with being soaked from rain and the like:
```suggestion
        return ret_val<bool>::make_failure( _( "This item cannot be used while underwater." ) );
```